backend/services/scheduler.go

```
@@ -10,82 +14,103 @@ import (
+
+			slog.Info("All parent jobs completed, scheduling job",
+				"jobId", job.DiggerJobID,
+				slog.Group("repository",
+					slog.String("fullName", repoFullName),
+					slog.String("owner", repoOwner),
+					slog.String("name", repoName),
+				),
+			)
+
 			ciBackend := ci_backends.GithubActionCi{Client: client}
 			ScheduleJob(ciBackend, repoFullName, repoOwner, repoName, batchId, job, gh)
```

golangci-lint (1.64.8)

Error return value is not checked

(errcheck)